lieer/remote.py

```diff
@@ -656,6 +644,15 @@ def __create_label__ (self, l):
         if excep.resp.status == 403 or excep.resp.status == 500:
           self.__request_done__ (False)
           return self.__create_label__ (l)
+        # not sure this is good enough, but it would be nice to be
+        # able to notify the end user that some specific tags were
+        # rejected by Gmail and then continue, without exiting with an
+        # exception.
+
+        # elif excep.resp.status == 400:
+        #   print("REJECTION: Label '{}' rejected by Gmail".format(l))
+        #   return (None, None)
```

Comment:
Unless we have a solution we should probably still do `raise` in the end

Reply:
I think we need to assume that we don't have exact information as for what are all the GMail's reserved labels (that, at least, is the impression I have) and also to assume that this can be changed in the future.
This is why I think that when a user's tag is rejected by GMail, we should just issue a warning, so the user can change the tag (or define a translation for it).
An exeption is surely a kind of warning, but I believe that end users should not get a exception in this case, and that a warning or an error message would be enough. But I'll the decision to you.
On the technical side, what I tried was to return a value that would let the program go on. (None, None) was causing errors, so perhaps you would know what is the right solution here, if we go that way.